plugin/PDFView.qml

```diff
@@ -144,6 +144,7 @@ SilicaFlickable {
 
         PinchArea {
             anchors.fill: parent
+            enabled: contextMenuLinks.opacity == 0. && contextMenuGoto.opacity == 0.
```

Comment:
Would be cleaner if stacking order handled this. Maybe fixed if context menu is on PDFDocumentPage.